google/cloud/spanner_v1/database.py

```diff
@@ -647,7 +676,7 @@ def execute_partitioned_dml(
             )
 
         def execute_pdml():
-            with SessionCheckout(self._pool) as session:
+            with SessionCheckout(self._pool, isLongRunning=True) as session:
```

Comment:
run_in_transaction also has a SessionCheckout call, should we not pass false over there?

Reply:
Also the session.transaction will be unset for run_in_transaction, so that needs to be added

Reply:
Default is False, so we don't need to pass. Also `run_in_transaction` is calling  `session.run_in_transaction` which is already setting the transaction . https://github.com/googleapis/python-spanner/blob/main/google/cloud/spanner_v1/session.py#L344